snap/snapcraft.yaml

```diff
@@ -124,7 +124,7 @@ parts:
         ;;
         ppc64le|s390x)
           # Cannot use alpine on ppc64le/s390x because it would require a musl agent
-          sudo -E PATH=$PATH make image DISTRO=ubuntu
```

Comment:
Hey!  Shouldn't you change the configuration file to actually use the initrd instead of the image?

Reply:
This is done automatically:

https://github.com/kata-containers/kata-containers/blob/6c39c353e603dda235d3b749b73b29203695bfff/snap/snapcraft.yaml#L171-L177